static/js/init.js

```diff
@@ -48,15 +48,15 @@ $(document).ready(function() {
     }
 
     // Forcing the .main-container to fit the entire height of the page
-    var setMainSize = function() {
-        var headerHeight = $('.site-header').height(),
-            footerHeight = $('footer').height(),
-            pageHeight = $('.page-wrap').height(),
-            mainHeight = pageHeight - headerHeight;
-            $('.main-container').height(mainHeight);
-    }
+    // var setMainSize = function() {
```

Comment:
I'm just not sure. I'm still figuring out how I'm doing the slide-out sides and this might be necessary. I'll remove it if we end up not needing it.